lua/telescope/builtin/init.lua

```diff
@@ -645,17 +645,40 @@ builtin.fd = builtin.find_files
 builtin.buffers = function(opts)
   opts = opts or {}
 
-  local buffers = filter(function(b)
-    return
-      (opts.show_all_buffers
-      or vim.api.nvim_buf_is_loaded(b))
-      and 1 == vim.fn.buflisted(b)
+  local buffers_list = vim.split(vim.fn.execute('buffers'), "\n")
+
+  local buffers = {}
+  opts.bufnr_width = 0
+  local default_selection_idx = 1
+  for i = 2, #buffers_list do
+    local bufnr, indicator, bufname, lnum = string.match(
```

Comment:
can we use `vim.api.nvim_list_bufs` instead or does it not do what you're looking for?

Reply:
also with `getbufinfo` or whatever that function is.

Reply:
looking into this tomorrow

Reply:
i used `:buffers` because i thought i can't get the indicators in another way. but i "compile" them now by myself. Thanks